GOVERNANCE.md

```diff
@@ -0,0 +1,216 @@
+
+The duties of a maintainer include the following: Maintainers are people who care about RIOT and want to help it grow and improve.
+A maintainer is not just someone who can make changes, but someone who has demonstrated their ability to collaborate with the team, get the most knowledgeable people to review code and docs, contribute high-quality code, and follow through to fix issues (in code or tests).
+More on maintaining RIOT can be found in the [maintaining guidelines].
+
+We are constantly looking for more maintainers.
+So if you are up for that, please start (or continue) contributing code and reviews!
+
+To contact maintainers, the best is to interact over actual RIOT code on [GitHub](https://github.com/RIOT-OS/RIOT/pulls).
+
+#### Becoming a Maintainer
+
+Maintainers can propose to give maintainer status to contributors that have been noticed as particularly active in some domain of RIOT.
+The decision to grant this status is then taken via consensus among the maintainers.
+If there is consensus on granting the status to a particular contributor, a maintainer will contact personally this contributor to propose the status, which the contributor can then accept (or turn down).
+
+Maintainers who are selected will be
+
+- invited to the [maintainers GitHub team] by one of the admins of the RIOT project, which grants them the necessary GitHub rights,
+- invited to the maintainer forum group by the forum moderators, which will give them access to the private maintainer forum, and
+- invited to the  private RIOT-maintainer chat room by one of the moderators of that room, for more informal exchanges between maintainers.
+
+#### Removing a Maintainer
+
+Maintainers may resign at any time if they feel that they will not be able to
+continue fulfilling their project duties.
+
+Maintainers may also be removed after being inactive, failure to fulfill their
+Maintainer responsibilities, violating the Code of Conduct, or other reasons.
+Inactivity is defined as a period of very low or no activity in the project.
+A yearly maintainer ping, an e-mail sent to inactive maintainers, determines if the maintainer is still willing to fulfill their project duties.
+On failure to reply to the maintainer ping within the specified amount of time (usually a month), the maintainer will be removed.
+
+### Release Managers
+
+Release managers make sure the quarterly release comes out in time.
+They are one or more maintainers which were appointed for a specific release by the Virtual Maintainer Assembly (VMA).
+Their duties include setting the dates for feature freeze for the release, enforcing the feature freeze, coordinating the (mostly automated) tests of a release, writing the release notes and creating the tags defining the release and its release candidates.
+The full set of tasks can be found in the document [Managing a Release].
+Their duties end, once the release is out or if all bug-fixing point releases are out.
```

Comment:
```suggestion
Their duties end once the release is out or if all bug-fixing point releases are out.
```

also, this is a bit unclear. How do we know that "all bug-fixing point releases are out"? Just before starting a new (soft/hard) feature freeze for the next release ?

Reply:
> also, this is a bit unclear. How do we know that "all bug-fixing point releases are out"? Just before starting a new (soft/hard) feature freeze for the next release ?

I think that is something we need to decide on. Because I don't know an answer to your questions.

Reply:
Moved to https://github.com/RIOT-OS/RIOT/pull/21067#discussion_r1878098710 as this got somewhat lost in GitHubs truncation.